tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -55,6 +60,7 @@ def email(self):
         return self.marionette.find_element(*self._email_locator).text
 
     def type_email(self, value):
+        self.marionette.find_element(*self._add_new_email_locator).tap()
         element = self.marionette.find_element(*self._email_locator)
```

Comment:
Perhapes this can be enclosed within a  Wait() statement, in case tap() takes some time?  It might cause intermittent failure.  Same for all below combination of .tap() followed by .find_element().